igvm_defs/src/lib.rs

```diff
@@ -1237,3 +1245,57 @@ pub enum VbsSigningAlgorithm {
     /// ECDSA P384.
     ECDSA_P384 = 0x1,
 }
+
+/// A structure defining a CoRIM CBOR payload for a given platform. TODO: rename
+/// to remove measurement?
+///
+/// The payload described by this header is a CBOR CoRIM payload. There may only
```

Comment:
Would be good to be have a similar level of detail here as below, pointing to section 4.1 of the spec, and explicitly calling it out as a tagged unsigned CoRIM map.